e2e/run.sh

```diff
@@ -1,3 +1,53 @@
 #!/bin/bash
 
-robot -P "${PWD}/libs" -L debug tests/replica_resiliency_test.robot
+usage () {
+   echo "USAGE: $0 --test_suite_path <specified_test_suite_path>  \ "
+   echo "          --test_case_name <specified_test_case_name> \ "
+   echo "Execute e2e testing with Robot Framework."
+   echo ""
+   echo "  -s, --test_suite_path             (Required) Test suite file path. e.g., /node_not_ready/node_reboot.robot or /replica_resiliency_test.robot"
```

Comment:
Currently, if we run `./run.sh -s /`, all test cases under the `/tests` folder will be executed. 

But I've just updated to support executing multiple specified test suites like `./run.sh -s <test-suite-1> -s <test-suite-2>`. Then it will execute all of test cases from `test-suite-1` and `test-suite-2`.

Reply:
Is the `-s` parameter necessary? Can't we by default include all test cases under the default folder? And filter what test cases we'd like to execute by `-t` parameter.

Reply:
I've updated as suggestion.
 - `-s`: optional, run all cases if empty
 - `-t`: optional, run specified test case